doc/source/whatsnew/v0.21.0.txt

```diff
@@ -34,6 +34,7 @@ Other Enhancements
 Backwards incompatible API changes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+- The ``DataFrame.style.to_excel()`` styled Excel export functionality now requires an external dependency, `cssdecl <http://pypi.python.org/pypi/cssdecl>`_.
```

Comment:
just add the PR number (this number) to the end of the whatsnew. & rebase. ping on green to merge.